spec/rspec/its_spec.rb

```diff
@@ -376,5 +378,58 @@ def terminator
         end
       end
     end
+
+    describe "#fits" do
+      it 'calls #its with :focus metadata' do
+        stub = Class.new do
+          include RSpec::Its
+        end.new
+        allow(stub).to receive(:its)
```

Comment:
Do you additionally need to `allow` when you `expect` below?

Reply:
Removed this spec